cheesed/cheezoid.py

```diff
@@ -1,3 +1,19 @@
+"""
+Copyright 2015 Ilya Sukhanov
```

Comment:
I couldn't figure out how to supply multiple contributors here. If anyone has ideas, I'm all ears.